modules/aws_postgres/tf_module/rds.tf

```diff
@@ -21,18 +21,20 @@ resource "aws_rds_cluster" "db_cluster" {
   count                   = var.existing_global_database_id == null ? 1 : 0
   cluster_identifier      = "opta-${var.layer_name}-${var.module_name}-${random_string.db_name_hash.result}"
   db_subnet_group_name    = "opta-${var.env_name}"
-  database_name           = var.database_name
+  database_name           = var.restore_from_snapshot == null ? var.database_name : null
```

Comment:
`database_name` is a valid parameter when restoring from a snapshot (see [`rds:RestoreDBClusterFromSnapshot`](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_RestoreDBClusterFromSnapshot.html), but [the docs](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_RestoreFromSnapshot.html) don't mention what happens when that is set.

Reply:
When it's not set then it inherits the original db's name, which I think is the behavior we want and I don't see a reason why we ought to support some renaming. It jus feels wrong and worse case scenario it can be done so via sql commands